index.html

```diff
@@ -56,9 +56,9 @@
 </h2>
 <ul class="triptych see-more clearfix">
   <li class="first-li">
-    <h3>My content isn’t code.</h3>
+    <h3>My project isn’t code.</h3>
```

Comment:
@benbalter With "content", you mean? "Project" feels more all-encompassing, but I don't feel strongly about it.